importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -2847,6 +2872,116 @@ static Marker* findMarker(const QString& repeat, Score* score)
       return m;
       }
 
+//---------------------------------------------------------
+//   isLikelyFingering
+//---------------------------------------------------------
+
+bool MusicXMLParserDirection::isLikelyFingering() const
+      {
+      // One or more newline-separated digits (or changes), possibly lead or trailed by whitespace
+      return _wordsText.contains(QRegularExpression("^\\s*[0-5pimac](?:[-–][0-5pimac])?(?:\\n[0-5pimac](?:[-–][0-5pimac])?)*\\s*$"))
+            && _rehearsalText == ""
+            && _metroText == ""
+            && _tpoSound < 0.1;
+      }
+
+//---------------------------------------------------------
+//   MusicXMLInferredFingering
+//---------------------------------------------------------
+
+MusicXMLInferredFingering::MusicXMLInferredFingering(qreal totalY,
+                                                     Element* element,
+                                                     QString text,
+                                                     int track,
+                                                     QString placement,
+                                                     Measure* measure,
+                                                     Fraction tick)
+      : _totalY(totalY),  _text(text), _element(element), _track(track), _placement(placement), _measure(measure), _tick(tick)
```

Comment:
Swapping  `_text(text)` and `_element(element)` resolves a compiler warning (and gets then into the order of the argument list). Maybe another addition for #8481 .

Reply:
Done!